files/aarch64_install_graphdb.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -euxo pipefail
+
+until ping -c 1 google.com &> /dev/null; do
+  echo "waiting for outbound connectivity"
+  sleep 5
+done
+
+timedatectl set-timezone UTC
+
+# 1. Install Tools
+#apt-get update -y 
+apt update && apt upgrade -y
+apt-get -o DPkg::Lock::Timeout=300 install -y unzip jq nvme-cli openjdk-11-jdk bash-completion
```

Comment:
I wonder if we could remove the apt cache in order to reduce the ami size (similar to what we do in the Docker builds) @yaskoo wdyt?

Reply:
I don't think it's worth it. This ami (disk) is around 4Gb, a few MB won't make much difference.